### PR TITLE
[OGM] Hibernate OGM 5.1.0.Final release post

### PR DESCRIPTION
**Pull request by DavideD**

### CODE REVIEW

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Good news!
+
+We released http://hibernate.org/ogm/[Hibernate OGM] 5.1 Final!
+
+Compared to the latest candidate release, there are not many changes. You can have a look at the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=22351[5.1.0.Final changelog]
+to get all the details.
+
+== What's the difference with 5.0 Final?
```
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: I'd rather say "What's new compared to version 5.0?"

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
+= "Hibernate OGM 5.1 Final is out"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Good news!
```
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: You already used this opening.. and I'm not sure if it is?

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* uses http://hibernate.org/orm/[Hibernate ORM] 5.1
+* supports Infinispan Remote via the http://infinispan.org/docs/stable/user_guide/user_guide.html#hot_rod_protocol[Hot Rod protocol]
+* supports Neo4j remote via the http://boltprotocol.org[Bolt] protocol and the http://neo4j.com/docs/rest-docs/current[Http interface]
+* uses Hibernate Search 5.6 (http://in.relation.to/2017/01/30/hibernate-search-5-6-0-Final-and-5-7-0-CR1[it works with Elasticsearch])
+* reduces the number of database calls by grouping the operations
+* supports https://docs.mongodb.com/manual/aggregation/#aggregation-pipeline[aggregation]
+  in MongoDB native queries with https://docs.jboss.org/hibernate/stable/ogm/reference/en-US/html_single/#ogm-mongodb-cli-syntax[CLI syntax]
+
+If you need to upgrade from a version before 5.1, you can find help on the
+https://developer.jboss.org/wiki/HibernateOGMMigrationNotes[migration notes].
```
Comment (yrodiere) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc:27]: There is nothing about 5.1 there, I guess you are about to add it?

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Compared to the latest candidate release, there are not many changes. You can have a look at the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=22351[5.1.0.Final changelog]
+to get all the details.
+
+== What's the difference with 5.0 Final?
+
+I'm glad you asked, this version:
+
+* uses http://hibernate.org/orm/[Hibernate ORM] 5.1
+* supports Infinispan Remote via the http://infinispan.org/docs/stable/user_guide/user_guide.html#hot_rod_protocol[Hot Rod protocol]
```
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: slightly better link: http://infinispan.org/docs/stable/user_guide/user_guide.html#using_hot_rod_server

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* supports Neo4j remote via the http://boltprotocol.org[Bolt] protocol and the http://neo4j.com/docs/rest-docs/current[Http interface]
+* uses Hibernate Search 5.6 (http://in.relation.to/2017/01/30/hibernate-search-5-6-0-Final-and-5-7-0-CR1[it works with Elasticsearch])
```
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: s/it works/, which works/

and

s/uses Hibernate Search/can integrate with Hibernate Search/

**Review by yrodiere — COMMENTED** [review]
Looks good to me, just one comment about the migration notes.

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Alternatively, you can download archives containing all the binaries, source code and documentation
+https://sourceforge.net/projects/hibernate/files/hibernate-ogm/5.1.0.Final[from Sourceforge].
+
+== What's next?
+
+The new features will include
+https://hibernate.atlassian.net/browse/OGM-75[custom types]
+and https://hibernate.atlassian.net/browse/OGM-470[the ability to expose unmapped
+attributes]. Also, Hibernate OGM 5.2 will work with Hibernate ORM 5.2.
```
Comment (gsmet) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: Is this really the roadmap we had in mind?

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+You can find us through the following channels:
+
+* https://forum.hibernate.org/viewforum.php?f=31[User forum]
+* https://hibernate.atlassian.net/browse/OGM[Issue tracker]
+* http://lists.jboss.org/pipermail/hibernate-dev/[Mailing list]
+* http://stackoverflow.com[Stack Overflow]: we monitor the tag _hibernate-ogm_
+* https://www.hipchat.com/gXEjW5Wgg[HipChat]: Hibernate OGM hipchat room
+
+We are looking forward to hear your feedback!
```
Comment (gsmet) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: to hearing

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 3 above)
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc:27]: https://hibernate.atlassian.net/browse/WEBSITE-231 ;-)

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 6 above)
Comment (Sanne) [inline posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: hum, no? Not according to the diagrams we made at the meeting. They schedule ORM 5.2 support for OGM 5.3

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 3 above)
Comment (DavideD) [reply posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc:27]: Yeah, I need to update it

**Review by Sanne — CHANGES_REQUESTED** [review]

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 6 above)
Comment (DavideD) [reply posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: Right, now that you mentioned it, I think I might have checked the wrong document

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 3 above)
Comment (DavideD) [reply posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc:27]: yeah, I've seen that :)

File: posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc (hunk 6 above)
Comment (DavideD) [reply posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: Can you remind the link to the document? I was using the one with the Agenda of the F2F meeting

Comment (Sanne) [reply posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc]: It's on the pictures. Can't share the link, they are private.

Anyway, the plans for OGM 5.2 include some improvements for the Hot Rod dialect, such as using batching. For ORM 5.2 that's at least OGM 5.3

**Comment by DavideD** [comment]
I've applied the remarks, anything else?

**Pull request closed without merging**